rpgJavaInterpreter-core/src/test/resources/db/WRITE02.rpgle

```diff
@@ -0,0 +1,23 @@
+     FEMPLVIEW  IF   E           K DISK
+     D EXPECTED        S              6
+      *--------------------------------------------------------------------------------------------*
+      *WRITING
+     C                   EVAL      EMPNO='000000'
+     C                   EVAL      FIRSTNME='JOHN'
+     C                   EVAL      MIDINIT='1'
+     C                   EVAL      LASTNAME='ROSS'
+     C                   EVAL      WORKDEPT='012'
+     C                   WRITE     EMPLOYEE
+     C                   EVAL      EMPNO='000001'
+     C                   EVAL      FIRSTNME='MARC'
+     C                   WRITE     EMPLOYEE
+      *--------------------------------------------------------------------------------------------*
+      *READING
+     C                   EVAL       EXPECTED = '000000'
+     C     KL            KLIST
```

Comment:
KLIST must be defined before assigning values to its fields

Reply:
I don't understand what it means.
Can you tell me where should I place the KLIST definition?
Sorry but I am not too strong in this topic...

Reply:
* Define KLIST to used
C     KL        KLIST
C                 KFLD                       EXPECTED
* Set value
C                 EVAL      EXPECTED = '000000'

Reply:
Done